### PR TITLE
Master

### PR DESCRIPTION
**Pull request by AshishSharma303**
RDS-Deployment-HA : this template is an addition of web and gateway server to existing RDS deployment. This template deploys VM's under azure LB and will join the new machines to Domain. This template will add new added VM's to existing RDS Farm.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@AshishSharma303__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@AshishSharma303__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: 301-vm-32-data-disks-high-iops/StoragePool.ps1/StoragePool.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Configuration StoragePool
```
Comment (singhkay) [inline 301-vm-32-data-disks-high-iops/StoragePool.ps1/StoragePool.ps1]: @AshishSharma303 What does this script add to 301-vm-32-data-disks-high-iops template?

File: rds-deployment-existing-HA/azuredeploy-HA.json [hunk 2]
```diff
@@ -0,0 +1,388 @@
+{
```
Comment (singhkay) [inline rds-deployment-existing-HA/azuredeploy-HA.json]: @AshishSharma303 Please see the contribution guide on the files needed for the PR to be merged https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE

**Comment by singhkay** [comment]
@AshishSharma303 Any update on this?

**Comment by gatneil** [comment]
No update in 4 months. CLosing

**Pull request closed without merging**